errors/parameter_errors.go

```diff
@@ -5,8 +5,10 @@ import (
 	"net/url"
 	"strings"
 
-	"github.com/pb33f/libopenapi-validator/helpers"
 	"github.com/pb33f/libopenapi/datamodel/high/base"
+
+	"github.com/pb33f/libopenapi-validator/helpers"
```

Comment:
There is reason to have a space between them for me

Here are some solutions, I'm suggesting

https://github.com/pb33f/libopenapi-validator/pull/101#discussion_r1832532789

Reply:
localmodule is working nicely